<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>618</width>
    <height>294</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: grey</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_6_">
    <property name="geometry">
     <rect>
      <x>202</x>
      <y>164</y>
      <width>92</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 10pt &quot;Bitstream Vera Sans&quot;;
background-color: rgb(35, 74, 148);</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_div">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>117</y>
      <width>93</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 10pt &quot;Bitstream Vera Sans&quot;;
background-color: rgb(35, 74, 148);</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5_">
    <property name="geometry">
     <rect>
      <x>103</x>
      <y>164</y>
      <width>93</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 10pt &quot;Bitstream Vera Sans&quot;;
background-color: rgb(35, 74, 148);</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_open_">
    <property name="geometry">
     <rect>
      <x>103</x>
      <y>70</y>
      <width>93</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 10pt &quot;Bitstream Vera Sans&quot;;
background-color: rgb(35, 74, 148);</string>
    </property>
    <property name="text">
     <string>(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>103</x>
      <y>211</y>
      <width>93</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 10pt &quot;Bitstream Vera Sans&quot;;
background-color: rgb(35, 74, 148);</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>202</x>
      <y>117</y>
      <width>92</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 10pt &quot;Bitstream Vera Sans&quot;;
background-color: rgb(35, 74, 148);</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_mult">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>164</y>
      <width>93</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 10pt &quot;Bitstream Vera Sans&quot;;
background-color: rgb(35, 74, 148);</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_point">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>258</y>
      <width>93</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 10pt &quot;Bitstream Vera Sans&quot;;
background-color: rgb(35, 74, 148);</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>211</y>
      <width>93</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 10pt &quot;Bitstream Vera Sans&quot;;
background-color: rgb(35, 74, 148);</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_close_">
    <property name="geometry">
     <rect>
      <x>202</x>
      <y>70</y>
      <width>92</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 10pt &quot;Bitstream Vera Sans&quot;;
background-color: rgb(35, 74, 148);</string>
    </property>
    <property name="text">
     <string>)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_qverty">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>243</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 10pt &quot;Bitstream Vera Sans&quot;;
background-color: rgb(35, 74, 148);</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>117</y>
      <width>93</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 10pt &quot;Bitstream Vera Sans&quot;;
background-color: rgb(35, 74, 148);</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4_">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>164</y>
      <width>93</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 10pt &quot;Bitstream Vera Sans&quot;;
background-color: rgb(35, 74, 148);</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_minus">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>211</y>
      <width>93</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 10pt &quot;Bitstream Vera Sans&quot;;
background-color: rgb(35, 74, 148);</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2_">
    <property name="geometry">
     <rect>
      <x>103</x>
      <y>117</y>
      <width>93</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 10pt &quot;Bitstream Vera Sans&quot;;
background-color: rgb(35, 74, 148);</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_0">
    <property name="geometry">
     <rect>
      <x>103</x>
      <y>258</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 10pt &quot;Bitstream Vera Sans&quot;;
background-color: rgb(35, 74, 148);</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_C">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>70</y>
      <width>93</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 10pt &quot;Bitstream Vera Sans&quot;;
background-color: rgb(35, 74, 148);</string>
    </property>
    <property name="text">
     <string>Clean</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>202</x>
      <y>211</y>
      <width>92</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 10pt &quot;Bitstream Vera Sans&quot;;
background-color: rgb(35, 74, 148);</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_plus">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>70</y>
      <width>93</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 10pt &quot;Bitstream Vera Sans&quot;;
background-color: rgb(35, 74, 148);</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(61, 56, 70);
border-color: rgb(97, 53, 131);
border: 2px;
font-size: 16px;
align-itrms: center;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_log">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>164</y>
      <width>93</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 10pt &quot;Bitstream Vera Sans&quot;;
background-color: rgb(35, 74, 148);</string>
    </property>
    <property name="text">
     <string>log</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_atan">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>258</y>
      <width>93</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 10pt &quot;Bitstream Vera Sans&quot;;
background-color: rgb(35, 74, 148);</string>
    </property>
    <property name="text">
     <string>atan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_mod">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>70</y>
      <width>93</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 10pt &quot;Bitstream Vera Sans&quot;;
background-color: rgb(35, 74, 148);</string>
    </property>
    <property name="text">
     <string>mod</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sqrt">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>211</y>
      <width>93</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 10pt &quot;Bitstream Vera Sans&quot;;
background-color: rgb(35, 74, 148);</string>
    </property>
    <property name="text">
     <string>sqrt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_pow">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>117</y>
      <width>93</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 10pt &quot;Bitstream Vera Sans&quot;;
background-color: rgb(35, 74, 148);</string>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_tan">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>164</y>
      <width>93</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 10pt &quot;Bitstream Vera Sans&quot;;
background-color: rgb(35, 74, 148);</string>
    </property>
    <property name="text">
     <string>tan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_asin">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>258</y>
      <width>93</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 10pt &quot;Bitstream Vera Sans&quot;;
background-color: rgb(35, 74, 148);</string>
    </property>
    <property name="text">
     <string>asin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_acos">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>211</y>
      <width>93</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 10pt &quot;Bitstream Vera Sans&quot;;
background-color: rgb(35, 74, 148);</string>
    </property>
    <property name="text">
     <string>acos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sin">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>117</y>
      <width>93</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 10pt &quot;Bitstream Vera Sans&quot;;
background-color: rgb(35, 74, 148);</string>
    </property>
    <property name="text">
     <string>sin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cos">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>20</y>
      <width>93</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 10pt &quot;Bitstream Vera Sans&quot;;
background-color: rgb(35, 74, 148);</string>
    </property>
    <property name="text">
     <string>cos</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>20</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(61, 56, 70);
border-color: rgb(97, 53, 131);
border: 2px;
font-size: 16;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="xlab">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>20</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
border: none;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ln">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>70</y>
      <width>93</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 10pt &quot;Bitstream Vera Sans&quot;;
background-color: rgb(35, 74, 148);</string>
    </property>
    <property name="text">
     <string>ln</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_X">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>20</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 10pt &quot;Bitstream Vera Sans&quot;;
background-color: rgb(35, 74, 148);</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_back">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>93</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 10pt &quot;Bitstream Vera Sans&quot;;
background-color: rgb(35, 74, 148);</string>
    </property>
    <property name="text">
     <string>backspase</string>
    </property>
   </widget>
   <widget class="QPushButton" name="main_lab">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
border: none;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>pushButton_6_</zorder>
   <zorder>pushButton_div</zorder>
   <zorder>pushButton_5_</zorder>
   <zorder>pushButton_open_</zorder>
   <zorder>pushButton_8</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>pushButton_mult</zorder>
   <zorder>pushButton_point</zorder>
   <zorder>pushButton_7</zorder>
   <zorder>pushButton_close_</zorder>
   <zorder>pushButton_qverty</zorder>
   <zorder>pushButton_1</zorder>
   <zorder>pushButton_4_</zorder>
   <zorder>pushButton_minus</zorder>
   <zorder>pushButton_2_</zorder>
   <zorder>pushButton_0</zorder>
   <zorder>pushButton_C</zorder>
   <zorder>pushButton_9</zorder>
   <zorder>pushButton_plus</zorder>
   <zorder>label</zorder>
   <zorder>pushButton_log</zorder>
   <zorder>pushButton_atan</zorder>
   <zorder>pushButton_mod</zorder>
   <zorder>pushButton_sqrt</zorder>
   <zorder>pushButton_pow</zorder>
   <zorder>pushButton_tan</zorder>
   <zorder>pushButton_asin</zorder>
   <zorder>pushButton_acos</zorder>
   <zorder>pushButton_sin</zorder>
   <zorder>pushButton_cos</zorder>
   <zorder>label_2</zorder>
   <zorder>pushButton_ln</zorder>
   <zorder>pushButton_X</zorder>
   <zorder>pushButton_back</zorder>
   <zorder>xlab</zorder>
   <zorder>main_lab</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
